brillig_vm/src/lib.rs

```diff
@@ -471,6 +464,7 @@ mod tests {
 
     #[test]
     fn cmp_binary_ops() {
+        let bit_size = 32;
```

Comment:
Previously this was not correct if both elements are modulo before the result (which I assume the intent is - to first cast to N bit integers)